index.html

```diff
@@ -35,38 +31,151 @@ <h1 class="greeting-hearder">I’m Sive <br> Glad to see you!</h1>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(1).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(2).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(3).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+    </section>
+    
+    <section class="about">
+        <h1 class="about-h1">About <br> Myself</h1>
+        <p class="info-summary">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="connect2"><h4>LET'S CONNECT</h4></a>
```

Comment:
- [ ] to be semantically correct, the `LET'S CONNECT` is a title describing the social media below it , kindly remove the anchor tag and leave it as h4 only 🙏